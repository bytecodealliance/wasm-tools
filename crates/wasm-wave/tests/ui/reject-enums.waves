keyword-cases-enum(true);
keyword-cases-enum(false);
keyword-cases-enum(some);
keyword-cases-enum(none);
keyword-cases-enum(ok);
keyword-cases-enum(err);
keyword-cases-enum(inf);
keyword-cases-enum(nan);