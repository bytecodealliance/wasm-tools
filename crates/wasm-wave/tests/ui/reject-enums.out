invalid value type at 19..23
invalid value type at 19..24
unexpected token: ParenClose at 23..24
invalid value type at 19..23
invalid value type at 19..21
invalid value type at 19..22
invalid value type at 19..22
invalid value type at 19..22
